staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -253,10 +255,11 @@ var (
 	)
 	apiserverRequestConcurrencyInUse = compbasemetrics.NewGaugeVec(
 		&compbasemetrics.GaugeOpts{
-			Namespace:         namespace,
-			Subsystem:         subsystem,
-			Name:              "request_concurrency_in_use",
-			Help:              "Concurrency (number of seats) occupied by the currently executing (initial stage for a WATCH, any stage otherwise) requests in the API Priority and Fairness subsystem",
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "request_concurrency_in_use",
+			Help:      "Concurrency (number of seats) occupied by the currently executing (initial stage for a WATCH, any stage otherwise) requests in the API Priority and Fairness subsystem",
+			// Remove this metric once all suppported releases have the equal current_executing_seats metric
```

Comment:
Similarly here

Reply:
The equal metric is being introduced in release 1.28, and that will be the oldest supported release when 1.31 is the latest release, so I am setting the deprecated version to 1.31 here.

Reply:
If we're following what we did for nominal_seats_limit, this should actually be 1.32 (4 releases from 1.28) since nominal_seats_limit was introduced in 1.26.

Reply:
We can fix in a follow-up though since this is a deprecation in a future version. I would be in favor of doing the deprecation on the same versions too, so maybe move both of them up to 1.32?